src/App.tsx

```diff
@@ -1,14 +1,47 @@
-/* eslint-disable max-len */
-import React from 'react';
-import 'bulma/css/bulma.css';
-import '@fortawesome/fontawesome-free/css/all.css';
+import {
+  FC,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 
-import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+
+import { TodoContext } from './context/ContextTodo';
+
+import { TodoFilter, getPreparedTodos } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { TodoList } from './components/TodoList';
 import { Loader } from './components/Loader';
 
-export const App: React.FC = () => {
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+export const App: FC = () => {
+  const {
+    setTodos,
+    isOpenModal,
+    todos,
+    inputField,
+    filteredBy,
+  } = useContext(TodoContext);
```

Comment:
Also here, you could create ur own hook like useTodos, so u won't need to type useContext(TodoContext) every time